### DISCUSSION TITLE
「建议」桌面小工具

### DISCUSSION THREAD

**Discussion by orange-juzipi** [linuxdeepin/developer-center#2835] [General | 常规讨论]
开发更多的桌面小工具或者插件，让用户能够更加便捷、更好的得到体验性。
1、比如说我目前是做Java开发的，然而我希望我能够用一款工具集成日常发开工作中所需的功能
2、还可以给大众用户带来一些日常使用中，经常用到的一些小工具，这样方便用户再去寻找半天而找不到。

小工具具有体验性高、便捷等用处。

**Comment by BLumia** [comment]
这建议也太宽泛了 :joy:

**Comment by Zeno-sole** [comment]
@OrangePeel-2019  你可以试一下utools

**Comment by meiyixiang** [comment]
可以具体一点。

**Comment by shenmo7192** [comment]
这种东西很容易由第三方提供的吧，自己写一个呗

**Comment by meiyixiang** [comment]
目前我们暂时没有这方面的计划，我感觉你想要的应该是一个可以管理常用工具的插件吧？
